### PR TITLE
Allow any string in close button

### PR DESCRIPTION
**Pull request by mkjems**
This will create another property called closeButtonContent that allows the user to define the content of the close button.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -54,6 +57,7 @@ var Dialog = createReactClass({
   getDefaultProps: function () {
     return {
       closeButtonLabel: 'Close this dialog window',
+      closeButtonContent: '\u00D7',
```
Comment (HugoGiraudel) [inline dist/index.js]: I believe this is not the same as `&times;`. Or am I mistaken?

**Comment by HugoGiraudel** [comment]
Wait, don’t close it? I’m willing to merge it.

File: dist/index.js (hunk 1 above)
Comment (mkjems) [inline dist/index.js]: I think it is :) 
http://www.fileformat.info/info/unicode/char/00d7/index.htm
(https://shripadk.github.io/react/docs/jsx-gotchas.html)

Comment (HugoGiraudel) [inline dist/index.js]: Oh, cool! Reopen that so I can merge it. :)

**Comment by HugoGiraudel** [comment]
Thanks! Released in 1.2.1: https://github.com/HugoGiraudel/react-a11y-dialog/releases/tag/1.2.1. :)

**Pull request merged**